kafka/librdkafka_vendor/rdkafka_mock.h

```diff
@@ -0,0 +1,331 @@
+/*
```

Comment:
What version/git-sha of librdkafka did you get this rdkafka_mock.h from?

Reply:
Hi,

i used the version which is referenced within the *.h file (1.8.2)
https://raw.githubusercontent.com/edenhill/librdkafka/v1.8.2/src/rdkafka_mock.h

Kristian